web/src/app/app.component.ts

```diff
@@ -8,7 +9,7 @@ import { MatIconRegistry } from '@angular/material/icon';
 import { AuthenticationService } from './core/services/authentication.service';
 import { ProfileModel } from './shared/models/index.model';
 import { Navigation } from './shared/models/navigation.model';
-import { SpinnerService } from './core/services/spinner.service';
+import { ActivityService } from './core/services/activity.service';
```

Comment:
Please follow alphabetical ordering in imports.

Reply:
Good spot, I will fix this 👍 .

I also created an issue to have a lint rule to highlight these https://github.com/DashboardHub/PipelineDashboard/issues/1242